reference/index.rst

```diff
@@ -0,0 +1,20 @@
+Reference
+=========
+
+* :doc:`/reference/annotation/index`
+
+* Configuration Formats
+
+    * :doc:`/reference/format/acl`
+    * :doc:`/reference/format/dashboard`
+    * :doc:`/reference/format/datagrid`
+    * :doc:`/reference/format/data_source`
+    * :doc:`/reference/format/entity_config`
+    * :doc:`/reference/format/navigation`
+    * :doc:`/reference/format/placeholders`
+    * :doc:`/reference/format/requirejs`
+
+.. toctree::
+    :maxdepth: 1
+
+    annotation/index
```

Comment:
We found problem on the orocrm site. To fix it you just need to add the following lines here:
entity_config/index
format/index